mountpoint-s3/src/inode.rs

```diff
@@ -170,7 +170,8 @@ impl Superblock {
                     match result {
                         Ok(HeadObjectResult { object, .. }) => {
                             let last_modified = object.last_modified;
-                            let stat = InodeStat::for_file(object.size as usize, last_modified, Instant::now());
+                            let etag = Some(object.etag);
```

Comment:
nit: I wouldn't bother with the separate line here, just write `Some(object.etag)` in the `for_file` call